vignettes/customise_query.Rmd

```diff
@@ -0,0 +1,409 @@
+
+
+```r
+hi <- oc_forward_df(placename = "Hawaii", no_annotations = FALSE)
+
+hi_bbox <-
+  oc_bbox(
+    hi$oc_southwest_lng,
+    hi$oc_southwest_lat,
+    hi$oc_northeast_lng,
+    hi$oc_northeast_lat
+  )
+
+oc_forward_df(placename = "Airport", bounds = hi_bbox, limit = 10)
+#> # A tibble: 10 x 4
+#>    placename oc_lat oc_lng oc_formatted                                                                                     
+#>    <chr>      <dbl>  <dbl> <chr>                                                                                            
+#>  1 Airport     21.3  -158. Daniel K. Inouye International Airport, Noonan Street, Honolulu, HI 96819, United States of Amer~
+#>  2 Airport     19.7  -156. Ellison Onizuka Kona International Airport at Keahole, Keahole Airport Road, Kailua-Kona, HI, Un~
+#>  3 Airport     20.9  -156. Kahului Airport, Hana Highway, Lower Paia, HI 96779, United States of America                    
+#>  4 Airport     19.7  -155. Hilo International Airport, Andrews Avenue, Hilo, HI 96720, United States of America             
+#>  5 Airport     21.4  -158. Kaneohe Bay Marine Corp Air Station/Mokapu Point Airport, C Street, Kailua, HI 96863, United Sta~
+#>  6 Airport     19.6  -156. Old Kona Airport State Recreation Area, Laniakea, Kailua-Kona, Hawaii, United States of America  
+#>  7 Airport     21.2  -157. Molokai Airport, Airport Loop, Kualapu‘u, HI 96729, United States of America                     
+#>  8 Airport     20.8  -157. Lanai Airport, Lanai Airport Road, Lanai City, HI 96763, United States of America                
+#>  9 Airport     21.3  -158. Kalaeloa Airport, Saratoga Avenue, Kapolei, HI 96862, United States of America                   
+#> 10 Airport     21.0  -157. Kapalua-West Maui Airport, Honoapiilani Highway, Napili, HI 96761, United States of America
+```
+
+Note that such a query will only give you aerodromes with "Airport" in their name or address, but not necessarily airfields, airstrips, etc. If you are more interested in these kind of features, you might want to take a look at the {[osmdata](https://docs.ropensci.org/osmdata/)} package.
+
+### `proximity`
+
+The `proximity` parameter provides OpenCage with a hint to bias results in favour of those closer to the specified location. 
+It is just one of many factors used for ranking results, however, and (some) results may be far away from the location or point passed to the `proximity` parameter. 
+A point is a named numeric vector of a latitude, longitude coordinate pair in decimal format. 
+The `proximity` parameter can most easily be specified with the `oc_points()` helper, for example like `proximity = oc_point(38.0, -84.5)` if you happen to already know the coordinates. If not you can also look them up with {opencage}, of course:
+
+
+```r
+lx <- oc_forward_df("Lexington, Kentucky")
+
+lx_point <- oc_points(lx$oc_lat, lx$oc_lng) 
+
+oc_forward_df(placename = "Paris", proximity = lx_point, limit = 5)
+#> # A tibble: 4 x 4
```

Comment:
btw for me tibble are 100% fine

Reply:
Yes, they are okay. What bothers me a bit, is that you cannot see the content of additional columns in wide tables.